plugins/loop_registry.go

```diff
@@ -0,0 +1,85 @@
+package plugins
+
+import (
+	"errors"
+	"sort"
+	"sync"
+)
+
+const (
+	PluginDefaultPort = 2112
```

Comment:
Is this only exported for test access? If so, could use a `_test.go` file to limit the export to `package plugins_test`.